mlcolvar/core/transform/descriptors/pairwise_distances.py

```diff
@@ -0,0 +1,124 @@
+import torch
+
+from mlcolvar.core.transform import Transform
+from mlcolvar.core.transform.descriptors.utils import compute_distances_matrix
+
+from typing import Union
+
+__all__ = ["PairwiseDistances"]
+
+class PairwiseDistances(Transform):
+    """
+    Pairwise distances transform, compute all the non duplicated pairwise distances for a set of atoms from their positions
+    """
+
+    def __init__(self, 
+                 n_atoms : int,
+                 PBC: bool,
+                 cell: Union[float, list],
+                 scaled_coords : bool = False,
+                 slicing_indeces : list = None) -> torch.Tensor:
```

Comment:
we could slice based on atom pairs like it is done in mdtraj 
https://www.mdtraj.org/1.9.8.dev0/api/generated/mdtraj.compute_distances.html#mdtraj.compute_distances

Reply:
I changed the slicing from number of the distance to the atom pairs list